docs/cloud-native-security/aws-securityhub.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[ingest-aws-securityhub-data]]
+= Ingest AWS Security Hub data
+
+In order to enrich your {elastic-sec} workflows with third-party cloud security posture data collected by AWS Security Hub:
+
+* Follow the steps to {integrations-docs}/aws/securityhub[set up the AWS Security Hub integration]. 
+
+* Make sure the integration version is at least 2.31.1. 
+
+* Ensure you have `read` privileges for the `security_solution-*.misconfiguration_latest` index.
+
+* While configuring the AWS Security Hub integration, turn on **Collect AWS Security Hub Findings from AWS**. We recommend you also set the **Initial Interval** value to `2160h` (equivalent to 90 days) to ingest existing logs.
+
+image::images/aws-config-finding-logs.png[AWS Security Hub integration settings showing the findings toggle]
+
+After you've completed these steps, AWS Security Hub data will appear on the <<cspm-findings-page, Misconfiguration Findings>> page. 
```

Comment:
Sort of a nit, but this sounds like the whole page is called "Misconfiguration Findings" which could mislead readers. More accurate, but wordier:

```suggestion
After you've completed these steps, AWS Security Hub data will appear on the Misconfigurations tab of the <<cspm-findings-page, Findings>> page. 
```

Or maybe something like:

```suggestion
After you've completed these steps, AWS Security Hub data will appear on <<cspm-findings-page, Findings | Misconfigurations>>. 
```